addons/email_confirmation/controllers.py

```diff
@@ -56,6 +56,33 @@ def web_auth_signup(self, *args, **kw):
 
         return request.render('auth_signup.signup', qcontext)
 
+    @http.route('/web/reset_password', type='http', auth='public', website=True)
+    def web_auth_reset_password(self, *args, **kw):
+        """Need to override, as all messages were originally in English."""
```

Comment:
It is not trivial to change one module's translation in another module. In fact I tried to do this once, and failed.